### DISCUSSION TITLE
Welcome to arrow-odbc-py Discussions!

### DISCUSSION THREAD

**Discussion by pacman82** [pacman82/arrow-odbc-py#44] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aersam** [comment]
Hi Markus!
I'm Adrian and we're using arrow-odbc to load lots of data from our on-prem sql server to the delta lake. arrow-odbc is a lot faster than what ADF or other tools can provide. 
Also, we're now using arrow-odbc in our [LakeAPI](https://github.com/bmsuisse/lakeapi) which is meant to be an API Layer on top of a delta lake and odbc. There we use arrow-odbc to also be able to deliver odbc data in the same way as delta lake tables, meaning we always need arrow record batches.
